ambari-server/src/main/resources/stack-hooks/before-START/scripts/params.py

```diff
@@ -266,14 +266,14 @@
 net_topology_mapping_data_file_path = os.path.join(net_topology_script_dir, net_topology_mapping_data_file_name)
 
 #Added logic to create /tmp and /user directory for HCFS stack.  
-has_core_site = 'core-site' in module_configs
+has_core_site = module_configs.get_all_properties(module_name, "core-site") is not None
```

Comment:
get_all_properties() never returns None. It returns an empty map if the config-type does not exist in the command.json

Reply:
You are right